app/Orchid/Screens/DonationEditScreen.php

```diff
@@ -88,7 +88,7 @@ public function layout(): array
              Layout::legend('donation', [
                 Sight::make('image')->render(function (Donation $donation) {
                     return "<img src={$donation['image']}
-                          alt='sample'
+                          alt='image-donation'
```

Comment:
is not necessary add "image" because the alt attribute belong to img etiquette (is redundant)